test/saithrift/src/switch_sai_rpc_server.cpp

```diff
@@ -737,7 +743,42 @@ class switch_sai_rpcHandler : virtual public switch_sai_rpcIf {
       free(attr_list);
       return status;
   }
-
+//listing all the fdb entries from map	
+  void sai_thrift_get_fdb_entries (sai_thrift_attribute_list_t& thrift_attr_list){
```

Comment:
I think this API should rather use the bulk get of FDB (or any object) provided from SAI.

Reply:
The get thrift API  for FDB is not defined, so  Prince (@prsunny ) suggested us to use this event notification method.

Reply:
Did you check saiobject.h 

sai_status_t sai_bulk_get_attribute(
        _In_ sai_object_id_t switch_id,
        _In_ sai_object_type_t object_type,
        _In_ uint32_t object_count,
        _In_ const sai_object_key_t *object_key,
        _Inout_ uint32_t *attr_count,
        _Inout_ sai_attribute_t **attr_list,
        _Inout_ sai_status_t *object_statuses);

Reply:
@atitjain , currently most SAI implementation doesn't have bulk get support though the API is available. As such, we also wanted to check the notification path and the stubs are being implemented as part of this PR. May be once the bulk get is available, we can add check for that as well.

Reply:
Sounds reasonable.